explainability-core/src/main/java/org/kie/trustyai/explainability/local/counterfactual/CounterfactualSolution.java

```diff
@@ -36,11 +36,10 @@
  */
 @PlanningSolution
 public class CounterfactualSolution {
-
+    @PlanningEntityCollectionProperty
     private List<CounterfactualEntity> entities;
     private List<Feature> originalFeatures;
 
-    @PlanningEntityCollectionProperty
     public List<CounterfactualEntity> getVaryingEntities() {
```

Comment:
Given that the entities are not accessed using `getVaryingEntities` anymore (and this doesn't seem to be used) could it be removed, perhaps?

Reply:
Yep, will do